workflows/zScanAction.yml

```diff
@@ -1,95 +1,60 @@
   pull_request:
-    branches: [master]
-  workflow_dispatch:
-    branches: [master]
-    
-jobs:
-  build:
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-    runs-on: ${{ matrix.os }}
-    steps:
-    - uses: actions/checkout@v3
-    - uses: actions/setup-java@v3
-      with:
-        distribution: temurin
-        java-version: 11
-        
-    - name: Setup Gradle
-      uses: gradle/gradle-build-action@v2
-      with:
-        gradle-version: 7.3.3
-    
-    - name: Execute Gradle build
-      run: ./gradlew build
+    branches: [ "main" ]
 
-    - name: Get apk path
-      id: apk-path
-      run: |
-        path=$(find **/build/outputs/apk -name '*.apk' -type f | head -1)
-        echo "::set-output name=path::$path"
-        
-    - name: Get apk info
-      id: apk-info
-      uses: hkusu/apk-info-action@v1
-      with:
-        apk-path: ${{ steps.apk-path.outputs.path }}
+permissions:
+  contents: read
 
-    - name: Move Artifact
-      uses: actions/upload-artifact@v3
-      with:
-        name: my-artifact
-        path: ${{ steps.apk-path.outputs.path }}
-        if-no-files-found: error # 'warn' or 'ignore' are also available, defaults to `warn`
-  
-  scan:
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-    runs-on: ${{ matrix.os }}
-    needs: build
+jobs:
+  zscan:
+    name: zScan
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read          # for actions/checkout to fetch code
+      security-events: write  # for github/codeql-action/upload-sarif to upload SARIF results
+      actions: read           # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v2
-
-    - name: Download application
-      uses: actions/download-artifact@v2
-      with:
-        name: my-artifact
-
-    - name: Zimperium
-      uses: zimperium/zscanmarketplace@v1
-      timeout-minutes: 60
-      with:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+  #  - name: Execute gradle build
+  #    run: echo "./gradlew build"
+      
+      - name: Run Zimperium zScan
+        uses: zimperium/zscanmarketplace@bfc6670f6648d796098c251ccefcfdb98983174d
+        timeout-minutes: 60
+        with:
         # REPLACE: Zimperium Client Environment Name
-        client_env: env_string
+          client_env: mapsfreemium
         # REPLACE: Zimperium Client ID
-        client_id: id_string
+          client_id: CNm4gbdCRIyIkv-yjUZ0_1
         # REPLACE: Zimperium Client Secret
-        client_secret: ${{ secrets.ZSCAN_CLIENT_SECRET }}
+          client_secret: ${{ secrets.ZSCAN_CLIENT_SECRET }}
         # REPLACE: The path to an .ipa or .apk
-        app_file: app-release-unsigned.apk
-
-    - name: Upload SARIF file
-      uses: github/codeql-action/upload-sarif@v2
-      with:
-        sarif_file: Zimperium.sarif
+          app_file: ./InsecureBankv2.apk
```

Comment:
maybe "REPLACEMENT-PATH/REPLACEMENT-APP.apk" would be a better placeholder name.